tests/eve_tests/definitions.py

```diff
@@ -295,6 +303,13 @@ def make_source_location(*, fixed: bool = False) -> SourceLocation:
     return SourceLocation(line=line, column=column, source=source)
 
 
+def make_source_location_group(*, fixed: bool = False) -> SourceLocation:
+    loc1 = make_source_location(fixed=fixed)
+    loc2 = make_source_location(fixed=fixed)
+
+    return SourceLocationGroup(loc1, loc2, context=make_str_value(fixed=fixed))
```

Comment:
Is it possible you meant the following?

```suggestion
def make_source_location_group(*, fixed: bool = False) -> SourceLocationGroup:
    loc1 = make_source_location(fixed=fixed)
    loc2 = make_source_location(fixed=fixed)

    return SourceLocationGroup(loc1, loc2, context=make_str_value(fixed=fixed))
```